### DISCUSSION TITLE
Сортировка Расчётных в Цикле

### DISCUSSION THREAD

**Discussion by Makhmut2445** [totumonline/totum-ru-issues-and-discussions#198] [Как это сделать?] [answered]
Расчетные в цикле не меняют свое расположение в зависимости от Сортировки в дереве... Это с чем связано?

**Comment by ttm-alexeypolunin** [comment, marked as answer]
При создании цикла `sort` таблиц записывается в `Версии таблиц в циклах` (2) и берется из `sort` таблицы в `Списке таблиц`.

В `Версиях таблиц` (1) `sort` тоже указан, но там он справочно — можно пересчитать строки с версиями и машина обновить `sort` из `Списка таблиц`.

<img width="586" alt="img_1" src="https://user-images.githubusercontent.com/55755565/186642213-7302e7b4-9255-456a-b124-52d1d7db2077.png">

Те если вы изменили `sort` в `Списке таблиц`, то вам нужно пойти в `Версии таблиц в циклах` и переставить там, что бы порядок обновился в уже созданных циклах (2  solid).

Расчет таблиц внутри цикла идет по `sort`. Но если вам нужно считать по одному порядку, а отображать по другому то за это отвечают поля (1 dashed) и (2 dashed):

<img width="999" alt="img_2" src="https://user-images.githubusercontent.com/55755565/186642984-c1180dce-8979-472e-b7e5-34288698778f.png">



Если вы хотите задать другой порядок отображения для создаваемых циклов то установите (1 dashed):

<img width="1514" alt="img_3" src="https://user-images.githubusercontent.com/55755565/186643074-1785f1f5-835d-4d76-99c7-54dc62a5b3f7.png">
